compat-data-schema.md

```diff
@@ -0,0 +1,374 @@
+`string` that contains a human-readable description of the sub-feature. As it is
+intended to be used as a kind of caption or title for the feature, keep it short.
+the `<code>` and `<a>`, as well as the macros `{{cssxref}}`, `{{HTMLElement}}`,
+`{{htmlattrxref}}`, and `{{domxref}}` can be used. See the localization section
+below for an explanation about how this string will be localized.
+* __Compat information__ contained in the `"support"` property. It contains an
+object listing the compat information for each browser. (See the description
+below.)
+* __Status information__ contained in the `"status"` object. It contains the
+information about the stability of the sub-feature: Is it a functionality that
+is standard? Is it stable? Has it been deprecated and shouldn't be used anymore
+(See the description below.)
+
+### The `"support"` object
+Each sub-feature has support information. For each browser identifier, it
+contains a compat object with the information about versions, prefixes or
+alternate names, as well as notes.
+
+The currently accepted browser identifiers are:
+* `"webview_android"`, Webview, the former stock browser on Android,
+* `"chrome"`, Google Chrome (on desktops),
+* `"chrome_android"`, Google Chrome (on Android),
+* `"edge"`, MS Edge (on Windows),
+* `"edge_mobile"`, MS Edge, the mobile version,
+* `"firefox`", Mozilla Firefox (on desktops),
+* `"firefox_android"`, Firefox for Android, sometimes nicknamed Fennec,
+* `"ie_mobile"`, Microsoft Internet Explorer, the mobile version,
+* `"ie"`, Microsoft Internet Explorer (discontinued)
+* `"opera"`, the Opera browser (desktop), based on Blink since Opera 15,
+* `"opera_android"`, the Opera browser (Android version)
+* `"safari"`, Apple Safari, on Mac OS,
+* `"safari_ios"`, Apple Safari, on iOS,
+* `"servo"`, the experimental Mozilla engine.
+
+No value is mandatory.
+
+Each of these properties contains a `support-statement` object with the
+practical compatibility information for this sub-feature and this browser.
+
+### The `support-statement` object
+This object is the key element of each browser compat information. It is a
+support-statement object. It is an array of `support` objects, but if there
+are only one of them, the array can be ommitted
+
+
+Example of an `support` compat object (with 2 entries):
+
+    {
+      "support": [
+        {
+          "version_added": "6.0"
+        },
+        {
+          "prefix": "-moz-",
+          "version_added": "3.5",
+          "version_removed": "9.0"
+        }
+
+      ]
+    }
+
+Example of a `support` compat object (with 1 entry, array ommitted):
+
+    {
+      "support": { "version_added": "6.0" }
+    }
+
+### Compat information in a `"support"` field.
+Compatibility information is stored in a `"support"` field. It may consist of the
+following properties:
+
+#### `"version_added"`
+Contains a string with the version number the sub-feature has
```

Comment:
I think it is an interesting idea: I quite like it, though I think it is may be complex to define and maintain. I think we should do this in a follow-up. Could you open an issue so that this is tracked?